features/networking/multus-ipv6.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Multus-CNI ipv6 related scenarios  
+  
+  # @author weliang@redhat.com
+  # @case_id OCP-28968
+  @admin
+  Scenario: IPv6 testing for OCP-21151: Create pods with multus-cni - macvlan bridge mode
+    # Make sure that the multus is enabled
+    Given the master version >= "4.1"
+    And the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    And evaluation of `node.name` is stored in the :target_node clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/IPv6/macvlan-bridge-v6.yaml" replacing paths:
```

Comment:
private repo

Reply:
@anuragthehatter Update to use <%= ENV['BUSHSLICER_HOME'] %>/testdata/networking/multus-cni/*. Thanks!